discovery/discovery.go

```diff
@@ -125,20 +129,21 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 			// Ignore not handled variables as {ext} isn't already rendered.
 			uri, _ := renderTemplate(m.uri, tplVars...)
 			sig, ok := renderTemplate(uri, "{ext}", "sig")
-			if ok {
-				de.Sig = append(de.Sig, sig)
+			if !ok {
```

Comment:
@jonboulle The behaviour isn't changed:
* The spec says (or better, It's inferred and I think it should be clarified, I'm going to open a PR on this) that a template (simple or meta) should contain an {ext} token. So if {ext} is present it will be rendered to "aci" and "sig", if not present the rendering will fail. So there isn't a possibility that the ACI url is rendered but not the Sig url.

* Given the above, it's the ACE that will download or not the signature and verify it or not. Actually rocket does this:
  * if -insecure-skip-verify=false (default) it tries do download also the sig and verify it
  * if -insecure-skip-verify=true it doesn't try to download the sig.